TestGeoIPMatcher6US fails with v2ray-geoip >= 202508050201

## What version of V2Ray are you using?

5.37.0/5.38.0


## What's your scenario of using V2Ray?

Packaging for a Linux distribution.


## What problems have you encountered?

The test suite fails at TestGeoIPMatcher6US:

```
=== RUN   TestGeoIPMatcher6US
    condition_geoip_test.go:194: expect US geoip contain 2001:4860:4860::8888, but actually not
--- FAIL: TestGeoIPMatcher6US (0.21s)
```

Downgrading v2ray-geoip to 202507050144 fixes it.

--- a/app/router/condition_geoip_test.go
+++ b/app/router/condition_geoip_test.go
@@ -18,13 +18,13 @@ import (
 )
 
 func init() {
-	const geoipURL = "https://raw.githubusercontent.com/v2fly/geoip/release/geoip.dat"
+	const geoipURL = "https://github.com/v2fly/geoip/releases/download/202507050144/geoip.dat"
 
 	wd, err := os.Getwd()
 	common.Must(err)
 
 	tempPath := filepath.Join(wd, "..", "..", "testing", "temp")
-	geoipPath := filepath.Join(tempPath, "geoip.dat")
+	geoipPath := filepath.Join(tempPath, "geoip-202507050144.dat")
 
 	os.Setenv("v2ray.location.asset", tempPath)
 
@@ -196,7 +196,7 @@ func TestGeoIPMatcher6US(t *testing.T) {
 }
 
 func loadGeoIP(country string) ([]*routercommon.CIDR, error) {
-	geoipBytes, err := filesystem.ReadAsset("geoip.dat")
+	geoipBytes, err := filesystem.ReadAsset("geoip-202507050144.dat")
 	if err != nil {
 		return nil, err
 	}

